### ISSUE TITLE
Add dhbwappendix environment

### ISSUE THREAD

**Issue by robingenz** [Skyfr3ak/dhbw-vs-latex-template#65]
## Description

Currently, the following code is needed to create an appendix section:
```
\newpage
\section*{Anhang X}\addcontentsline{toc}{section}{Anhang X}\label{sec:anhang_X}
```

## Recommended solution

A "dhbwappendix" environment would be helpful to facilitate this step.

## Alternative solutions

Add a code snippet for VS Code.

**Comment by robingenz** [comment]
There should also be an additional [label tag](https://github.com/skyfrk/dhbw-vs-latex-template/blob/master/docs/writing.md#labels-and-referencing).
Example for current behavior with `sec` (`\label{sec:anhang_1}`): "Ein Beispiel befindet sich in Abschnitt 5."
Correct behavior: "Ein Beispiel befindet sich in Anhang 1."

@skyfrk  Should I create an additional issue for that?

**Comment by skyfrk** [comment]
@robingenz In order to archive this we have to read the docs of the `\autoref{}` command and figure out how to add a custom tag. Feel free to go ahead and create another issue!

**Comment by skyfrk** [comment]
I'll release a fix for this issue and #67 this night / tomorrow

`appendix` CTAN Package 🙏
